.github/workflows/appsignals-e2e-test.yml

```diff
@@ -78,17 +85,18 @@ jobs:
           --approve
 
       - name: Set up terraform
-        uses: hashicorp/setup-terraform@v2
+        uses: hashicorp/setup-terraform@v3
         with:
           terraform_wrapper: false
 
       - name: Deploy sample app via terraform
-        working-directory: test/terraform/eks
+        working-directory: testing/terraform/eks
```

Comment:
Why are we changing the directory?

Reply:
We're pulling the testing resources from ADOT directly now, where the folder is named `testing`. We didn't get to choose the names because each repo already had a preexisting `test` or `testing` folder.

Reply:
Makes sense.